packages/components/src/font-size-picker/stories/index.tsx

```diff
@@ -8,11 +13,10 @@ import { useState } from '@wordpress/element';
  */
 import FontSizePicker from '../';
 
-export default {
+const meta: ComponentMeta< typeof FontSizePicker > = {
```

Comment:
With the FontSizePicker having its own types, we should be able to delete most of the type defs under the `argTypes` property (just not sure about `withReset` since it seems to have some advanced config)

Reply:
Nice 👍